client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx

```diff
@@ -91,7 +92,7 @@ const PlansWrapper: React.FC< Props > = ( props ) => {
 		? reduxHideFreePlan && 'plans-blog-onboarding' === plansIntent
 		: reduxHideFreePlan;
 
-	const onSelectPlan = ( cartItems?: MinimalRequestCartProduct[] | null ) => {
+	const onUpgradeClick = ( cartItems?: MinimalRequestCartProduct[] | null ) => {
```

Comment:
**Note:**

I switch the name from `onSelectPlan` to `onUpgradeClick` because we will now be processing both a plan cart item _and_ a storage add-on cart item with this function